core/pkg/resourcehandler/k8sresources.go

```diff
@@ -147,6 +148,29 @@ func (k8sHandler *K8sResourceHandler) GetClusterAPIServerInfo() *version.Info {
 	return clusterAPIServerInfo
 }
 
+// set  namespaceToNumOfResources map in report
+func setMapNamespaceToNumOfResources(allResources map[string]workloadinterface.IMetadata, sessionObj *cautils.OPASessionObj) {
+	mapNamespaceToNumberOfResources := make(map[string]int)
+	for _, resource := range allResources {
+		if obj := workloadinterface.NewWorkloadObj(resource.GetObject()); obj != nil {
+			ownerReferences, err := obj.GetOwnerReferences()
+			if err == nil {
+				// if object is highest level and belong to namespace (except Job), add to map
```

Comment:
1. :fix: `// if object is highest level and belong to namespace (except Job), add to map` -> `// Add an object to the map if the object does not have a parent but is contained within a namespace (except Job)`
2. :question: Why are we excluding jobs?

Reply:
since they run only once and when showing in UI they have pbbly died

Reply:
Is this a requirement?

Reply:
yes